### DISCUSSION TITLE
Why is a 302 (temporary redirect) on first use of instance when using a search query

### DISCUSSION THREAD

**Discussion by MadAim123** [searxng/searxng#3219] [Q&A]
When I open an [url like this](https://searxng.site/searxng/search?q=test&language=de&safesearch=0&categories=general&time_range=month) I'm redirected with 302 status to the startpage. Even with preference in url.
Why is this temporary redirct on almost all instances and how to get out of this? Can I change a prameter in my URL or how to resolve it?

**Comment by return42** [comment]
Are you using a terminal browser or a command like `curl`? / then it is the bot detection ..

**Reply by MadAim123** [reply]
Desktop browser. With Firefox, Chrome... I tried it all.

I also tried it with python request and used headers and so on to not be seen as a bot.

the bot detection would also end in status 429 (too many requests)

**Reply by return42** [reply]
Then I don't know what your issue is / I can open https://searxng.site/searxng/search?q=test&language=de&safesearch=0&categories=general&time_range=month

**Reply by MadAim123** [reply]
When I open it for the second time in the same browser, it works for me too.
But with another Instance (https://searx.space/) I have the same problem.

If an Instance is not working, I would like to switch to another instance but It's always the same problem. 302 redirection on first use and I would get to know, why.

**Reply by MadAim123** [reply]
Now I found an issue with the same problem: https://github.com/searxng/searxng/issues/3191
